Makefile

```diff
@@ -1,7 +1,7 @@
 PATH := $(CURDIR)/build-tools/bin:$(PATH)
 
 # How large to we want the disk to be in Mb
-MEDIA_SIZE=700
+MEDIA_SIZE=8192
```

Comment:
I think it would be a bit cleaner to just tweak it for the run-installer target and leave the default value be. I think simply adding the following line right after MEDIA_SIZE=700 will do the trick:

run-installer: MEDIA_SIZE=8192

Reply:
Given that ppal wanted to install applications using fallback.img (which failed due to not sufficient space) it seems useful to apply this across the board.